logd/src/main/resources/configdefinitions/logd.def

```diff
@@ -7,10 +7,13 @@ stateport int default=0
 ## Host to contact the logserver on.
 logserver.host string default="localhost"
 
-## Port to contact the logserver on.
-logserver.port int default=5821
+## RPC port of logserver.
+logserver.rpcPort int default=5821
```

Comment:
I would name this rpcport to align with naming elsewhere in the file (and def files used only in cpp code)